app/forms/journeys/additional_payments_for_teaching/poor_performance_form.rb

```diff
@@ -10,7 +10,8 @@ class PoorPerformanceForm < Form
       def save
         return false unless valid?
 
-        update!(eligibility_attributes: attributes)
+        update!(eligibility_attributes: attributes.except("journey_session_id"))
+        journey_session.update(answers: attributes.except("journey_session_id"))
```

Comment:
Another option would be something like:
```
journey_session.update(attributes.except("journey_session_id"))
```

So we don't need to worry about merging over overwriting in the client. We just need a generic setter in `Journeys::Session` which handles it.